### PR TITLE
Revise TLS docs for InfluxDB OSS v1

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: content/influxdb/v1.7/administration/https_setup.md [hunk 1]
```diff
@@ -8,21 +8,31 @@ menu:
     parent: Administration
 ---
 
-Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
-When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
+Enabling TLS encrypts the communication between clients and the InfluxDB server.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: Enable TLS to encrypt communication...

File: content/influxdb/v1.7/administration/https_setup.md [hunk 2]
```diff
@@ -8,21 +8,31 @@ menu:
...
+When configured with a signed certificate, TLS also allows clients to verify the authenticity of the InfluxDB server.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: Configure TLS? w a signed certificate to allow a client....

File: content/influxdb/v1.7/administration/https_setup.md [hunk 3]
```diff
@@ -8,21 +8,31 @@ menu:
...
 
-InfluxData [strongly recommends](/influxdb/v1.7/administration/security/) enabling HTTPS, especially if you plan on sending requests to InfluxDB over a network.
+To set up TLS over HTTPS, do the following:
 
-## Requirements
+- [Obtain requirements](#requirements)
+- [Configure InfluxDB to use TLS](#configure-influxdb-to-use-tls)
+
+{{% warn %}}
+InfluxData **strongly recommends** enabling HTTPS, especially if you plan on sending requests to InfluxDB over a network.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: plan to send

File: content/influxdb/v1.7/administration/https_setup.md [hunk 4]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
 
 * **Self-signed certificates**
 
-    Self-signed certificates are _not_ signed by a Certificate Authority (CA).
-    [Generate a self-signed certificate](#step-1-generate-a-self-signed-certificate) on your own machine.
+    Self-signed certificates are _not_ signed by a trusted, third-party CA.
     Unlike CA-signed certificates, self-signed certificates only provide cryptographic security to HTTPS requests.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: Self-signed certificates provide...but don't allow clients....

File: content/influxdb/v1.7/administration/https_setup.md [hunk 5]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
     They do not allow clients to verify the identity of the InfluxDB server.
     With this certificate option, every InfluxDB instance requires a unique self-signed certificate.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: what certificate option?

File: content/influxdb/v1.7/administration/https_setup.md [hunk 6]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
 
-  # Use a separate private key location.
-  https-private-key = "<bundled-certificate-file>.pem"
-```
-
-#### Step 5: Restart the InfluxDB service
+## Configure InfluxDB to use TLS
 
-Restart the InfluxDB process for the configuration changes to take effect:
+1. **Download or generate certificate files**
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: ...a certificate file?

File: content/influxdb/v1.7/administration/https_setup.md [hunk 7]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
 
-```bash
-sudo systemctl restart influxdb
-```
+    If using a certificate provided by a CA, follow their instructions to download the certificate files.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: ...using a CA cert...

File: content/influxdb/v1.7/administration/https_setup.md [hunk 8]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
 
-#### Step 6: Verify the HTTPS setup
+    If using a self-signed certificate, use the `openssl` utility to create a certificate.
 
-Verify that HTTPS is working by connecting to InfluxDB with the [CLI tool](/influxdb/v1.7/tools/shell/):
+    The following command generates a private key file (.key) and a self-signed certificate file (.crt) with required permissions
+    and saves them to `/etc/ssl/`.
+    (Other paths will also work.)
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: Use the following command to generate...and save the files to the specified path (by default, `etc...).

File: content/influxdb/v1.7/administration/https_setup.md [hunk 9]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
+    Files remain valid for the specified `NUMBER_OF_DAYS`.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: Set `NUM....to specify...

File: content/influxdb/v1.7/administration/https_setup.md [hunk 10]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
 
-A successful connection returns the following:
-
-```bash
-Connected to https://<domain_name>.com:8086 version 1.x.x
-InfluxDB shell version: 1.x.x
->
-```
+    The command will prompt you for more information.
+    You can choose to fill out these fields or leave them blank; both actions generate valid certificate files.
```
Comment (kelseiv) [inline content/influxdb/v1.7/administration/https_setup.md]: Follow prompts to set values (or leave blank to set default values).

Think "generate valid cert files" is implied?

File: content/influxdb/v1.7/administration/https_setup.md [hunk 11]
```diff
@@ -8,21 +8,31 @@ menu:
...
-Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
-When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
+Enabling TLS encrypts the communication between clients and the InfluxDB server.
+When configured with a signed certificate, TLS also allows clients to verify the authenticity of the InfluxDB server.
 
-InfluxData [strongly recommends](/influxdb/v1.7/administration/security/) enabling HTTPS, especially if you plan on sending requests to InfluxDB over a network.
+To set up TLS over HTTPS, do the following:
 
-## Requirements
+- [Obtain requirements](#requirements)
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/https_setup.md]: This seems like an odd step to have in the documentation.  Why not just list the requirements here:

> To set up TLS over HTTPS, you need the following: ...

File: content/influxdb/v1.7/administration/https_setup.md [hunk 12]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
-
-  # Determines whether HTTPS is enabled.
-  https-enabled = true
-
-  [...]
-
-  # The SSL certificate to use when HTTPS is enabled.
-  https-certificate = "<bundled-certificate-file>.pem"
+<!-- InfluxDB supports certificates composed of a private key file (`.key`) and a signed certificate file (`.crt`) file pair, -->
+<!-- as well as certificates that combine the private key file and the signed certificate file into a single bundled file (`.pem`). -->
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/https_setup.md]: Is there a reason these are commented out?

File: content/influxdb/v1.7/administration/https_setup.md (hunk 6 above)
Comment (sanderson) [reply content/influxdb/v1.7/administration/https_setup.md]: @kelseiv "certificate" actually refers to 3 separate things: a client certificate, a certificate authority (certifies the validity of the certificate), and a private key. These can be separate files or contained in a single file (`.pem`).

File: content/influxdb/v1.7/administration/https_setup.md [hunk 13]
```diff
@@ -32,180 +42,98 @@ InfluxDB supports three types of TLS certificates:
...
-```bash
-sudo systemctl restart influxdb
-```
+    If using a certificate provided by a CA, follow their instructions to download the certificate files.
 
-#### Step 6: Verify the HTTPS setup
+    If using a self-signed certificate, use the `openssl` utility to create a certificate.
 
-Verify that HTTPS is working by connecting to InfluxDB with the [CLI tool](/influxdb/v1.7/tools/shell/):
+    The following command generates a private key file (.key) and a self-signed certificate file (.crt) with required permissions
```
Comment (sanderson) [inline content/influxdb/v1.7/administration/https_setup.md]: Format ".key" and ".crt" as inline code.

**Pull request merged**